server/routes/dsl.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import {
+  IRouter,
+} from '../../../../src/core/server';
+import { schema } from '@osd/config-schema';
+import DSLFacet from '../services/facets/dslFacet';
+import {
+  DSL_BASE,
+  DSL_SEARCH,
+  DSL_CAT,
+  DSL_MAPPING
+} from '../../common/constants/shared';
+import { RequestParams } from '@elastic/elasticsearch';
```

Comment:
Just checking, we don't have any preferences against using imports from `elastic`, I assume?

Reply:
it's only available in the `@elastic/elasticsearch` package so it should be fine